commands/audit/sca/python/python.go

```diff
@@ -103,9 +115,63 @@ func getDependencies(auditPython *AuditPython) (dependenciesGraph map[string][]s
 		sca.LogExecutableVersion("python")
 		sca.LogExecutableVersion(string(auditPython.Tool))
 	}
+	if auditPython.IsCurationCmd {
+		pipUrls, errProcessed := processPipDownloadsUrlsFromReportFile()
+		if errProcessed != nil {
+			err = errProcessed
+			return
+		}
+		params.SetDownloadUrls(pipUrls)
+	}
 	return
 }
 
+func processPipDownloadsUrlsFromReportFile() (map[string]string, error) {
+	exist, err := fileutils.IsFileExists("report.json", false)
+	if err != nil {
+		return nil, err
+	}
+	if !exist {
+		err = errors.New("process failed, report file wasn't found, cant processed with curation command")
+		return nil, err
+	}
+	var reportBytes []byte
+	reportBytes, err = fileutils.ReadFile("report.json")
+	if err != nil {
+		return nil, err
+	}
+	pipReport := &pypiReport{}
+	if err = json.Unmarshal(reportBytes, pipReport); err != nil {
+		return nil, err
+	}
+	pipUrls := map[string]string{}
+	for _, dep := range pipReport.Install {
+		if dep.MetaData.Name != "" {
+			compId := PythonPackageTypeIdentifier + strings.ToLower(dep.MetaData.Name) + ":" + dep.MetaData.Version
+			pipUrls[compId] = strings.Replace(dep.DownloadInfo.Url, "api/curation/audit/", "", 1)
```

Comment:
add tests for this operation (add report.json file / run the command for the test project and compare results)